q2.html

```diff
@@ -0,0 +1,94 @@
+    td, th {
+        border: 1px solid #dddddd;
+        text-align: left;
+        padding: 8px;
+      }
+      tr:nth-child(even) {
+        background-color: black;
+        color:white;
+      }
+</style>
+</head>
+<body>
+    <div id="mainform" style="background-color:black;color:white;border-radius:10px;border: 1px solid black; padding: 20px;width:320px;margin:auto;display:flex;justify-content:space-evenly">
+        <form id="myForm">
+            <div>
+                <label for="name">Name:</label>
+                <input type="text" id="name" name="name"></br></br>
+                <label for="email">Email:</label>
+                <input type="text" id="email" name="email"></br></br>
+                <!-- Radio For Gender -->
+                <label for="gender">Gender:</label>
+                <input type="radio" id="male" name="gender" value="male">
+                <label for="male">Male</label>
+                <input type="radio" id="female" name="gender" value="female">
+                <label for="female">Female</label><br></br>
+
+                <label for="age">Age:</label>
+                <input type="number" id="age" name="age"></br></br>
+
+                 <!-- CheckBox For Hobbies -->
+                <label for="hobbies">Hobbies:</label>
+                <input type="checkbox" id="hobbies1" name="hobbies" value="Reading">
+                <label for="hobbies1"> Reading</label>
+                <input type="checkbox" id="hobbies2" name="hobbies" value="Travelling">
+                <label for="hobbies2"> Travelling</label>
+                <input type="checkbox" id="hobbies3" name="hobbies" value="Sports">
+                <label for="hobbies3"> Sports</label></br></br>
+
+                <label for="country">Country:</label>
+                <select id="country" name="country">
+                    <option value="india">India</option>
+                    <option value="usa">USA</option>
+                    <option value="russia">Russia</option>
+                 </select></br></br>
+    
+                <label for="state">State:</label>
+                <select id="state" name="state">
+                    <option value="gujarat">Gujarat</option>
+                    <option value="dehli">Dehli</option>
+                    <option value="up">UP</option>
+                 </select></br></br>
+
+                <label for="city">City:</label>
+                <select id="city" name="city">
+                    <option value="surat">Surat</option>
+                    <option value="ahmedabad">Ahmedabad</option>
+                    <option value="bharuch">Bharuch</option>
+                 </select></br></br>
+                <div style = "
+                        border-radius: 65%;
+                        border-background: black;
+                        display: flex;
+                        justify-content: space-evenly;">
+                        <button id= "button1" type="button">Submit</button>
```

Comment:
Format the document properly. I can see here there is no space before `=` in `id= `. Either give space to right and left both sides either don't give.

Reply:
OKay Sir